packages/backend-deployer/src/cdk_error_mapper.ts

```diff
@@ -79,14 +92,17 @@ export class CdkErrorMapper {
         /Invalid AttributeDataType input, consider using the provided AttributeDataType enum/,
       humanReadableErrorMessage:
         'User pool attributes cannot be changed after a user pool has been created.',
+      resolutionMessage:
+        'To change these attributes, remove `defineAuth` from your backend, then add it back.',
```

Comment:
should we be recommending this without some warnings on the side-effects? like what it means for the auth resources

Reply:
good call. Updated the message.